protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -110,84 +110,84 @@ Select {
+4. Create ethereum oracles based on calling a read method of a smart contract by supplying incorrect ABI (Phase 2 - oracle based on listening for events) (<a name="0082-ETHD-004" href="#0082-ETHD-004">0082-ETHD-004</a>)
 
 ### External Oracles - Amendments
 
-1. Amend the oracle data source via governance proposals. Amendments should take effect as soon as the proposal is enacted (<a name="0081-ETHD-005" href="#0081-ETHD-005">0081-ETHD-005</a>)
-2. Amend an existing ethereum data source and change the contract address and enact the proposal. Once enacted, the data should be sourced from the new smart contract. Try amending the other fields in the oracle data source and the changes should take effect after the proposals are enacted (<a name="0081-ETHD-006" href="#0081-ETHD-006">0081-ETHD-006</a>)
-3. Phase 2 - Amend an existing ethereum data source and change the events that we are listening to and enact the proposal. Once enacted , the data should be sourced from the amended events. Try amending the other fields in the oracle data source and the changes should take effect after the proposals are enacted (<a name="0081-ETHD-007" href="#0081-ETHD-007">0081-ETHD-007</a>)
+1. Amend the oracle data source via governance proposals. Amendments should take effect as soon as the proposal is enacted (<a name="0082-ETHD-005" href="#0082-ETHD-005">0082-ETHD-005</a>)
+2. Amend an existing ethereum data source and change the contract address and enact the proposal. Once enacted, the data should be sourced from the new smart contract. Try amending the other fields in the oracle data source and the changes should take effect after the proposals are enacted (<a name="0082-ETHD-006" href="#0082-ETHD-006">0082-ETHD-006</a>)
+3. Phase 2 - Amend an existing ethereum data source and change the events that we are listening to and enact the proposal. Once enacted , the data should be sourced from the amended events. Try amending the other fields in the oracle data source and the changes should take effect after the proposals are enacted (<a name="0082-ETHD-007" href="#0082-ETHD-007">0082-ETHD-007</a>)
 
 ### External Oracles - Deletions
 
-1. Ability to delete a data source if and only if the data source is NOT used by any active markets (<a name="0081-ETHD-008" href="#0081-ETHD-008">0081-ETHD-008</a>)
-2. Should NOT be able to delete a data source if its being actively used by a market (<a name="0081-ETHD-009" href="#0081-ETHD-009">0081-ETHD-009</a>)
-3. If a single data source is used by multiple markets, then should NOT be able to delete the data source even if one of those markets is actively using the data source (<a name="0081-ETHD-010" href="#0081-ETHD-010">0081-ETHD-010</a>)
+1. Ability to delete a data source if and only if the data source is NOT used by any active markets (<a name="0082-ETHD-008" href="#0082-ETHD-008">0082-ETHD-008</a>)
```

Comment:
Can you explain the intended mechanics for creating/deleting data sources if it does not happen automatically when they are first reference (create) and no longer reference (delete)? If it is automatic I think the AC could be made clearer.

Reply:
The intended mechanism to deactivate and delete a data source is the one we have at the moment.
Seems a little rewording will help, I'll suggest changes.